src/ontology/external/processed-mondo-omim-genes.robot.tsv

```diff
@@ -44,7 +44,6 @@ http://purl.obolibrary.org/obo/MONDO_0007163	http://identifiers.org/hgnc/1388	OM
 http://purl.obolibrary.org/obo/MONDO_0007164	http://identifiers.org/hgnc/12642	OMIM:108600	https://omim.org/entry/185880
 http://purl.obolibrary.org/obo/MONDO_0007167	http://identifiers.org/hgnc/3755	OMIM:108720	https://omim.org/entry/603381
 http://purl.obolibrary.org/obo/MONDO_0007168	http://identifiers.org/hgnc/3755	OMIM:108721	https://omim.org/entry/603381
-http://purl.obolibrary.org/obo/MONDO_0007171	http://identifiers.org/hgnc/4279	OMIM:108770	https://omim.org/entry/121013
```

Comment:
Overall, the issues in this file mirror the issues in `src/ontology/external/mondo-omim-genes.robot.tsv`